templates/schema.sql

```diff
@@ -0,0 +1,45 @@
+CREATE DATABASE auctionshop
+DEFAULT CHARACTER SET utf8
+DEFAULT COLLATE utf8_general_ci;
+
+use auctionshop;
+
+CREATE TABLE categories(
+    `id` INT PRIMARY KEY,
+    `name` VARCHAR(255) NOT NULL,
+    `technical_name` VARCHAR(64) NOT NULL
+);
+
+CREATE TABLE users(
+    `id` INT PRIMARY KEY,
+    `registrated_at` DATE DEFAULT CURRENT_DATE,
+    `email` VARCHAR(255) NOT NULL,
```

Comment:
В RFC вроде 320 максимально указано, но это уже лучше, чем было раньше)
Можем оставить так, просто было сказано к тому, что столбцы надо проектировать с оглядкой на стандарты

Reply:
Я прочитала, что максимум 256. 
Не вижу информацию о 320
Since addresses
   that do not fit in those fields are not normally useful, the upper
   limit on address lengths should normally be considered to be 256

Reply:
> That limit is a maximum of 64 characters (octets)
>    in the "local part" (before the "@") and a maximum of 255 characters
>    (octets) in the domain part (after the "@") for a tot
al length of 320
>    characters.That limit is a maximum of 64 characters (octets)
   in the "local part" (before the "@") and a maximum of 255 characters
   (octets) in the domain part (after the "@") for a total length of 320
   characters.
   
   Есть такое, но да, соглашусь, дальше сказано что хватит 256